.github/workflows/lint.yml

```
@@ -23,13 +23,13 @@ jobs:
               run: npm run prettier:check
             - name: ESLint
               if: success() || failure()
-              run: npm run eslint -- --max-warnings 0 || false
+              uses: forsakringskassan/eslint-config@feature/github-action
```

```suggestion
              uses: forsakringskassan/eslint-config@main
```

(eller om vi vill versionshantera så `@v1`)